generate_testdata.sh

```diff
@@ -65,6 +65,7 @@ scaffold_test_project() {
             $kb create webhook --group crew --version v1 --kind FirstMate --conversion
             $kb create api --group crew --version v1 --kind Admiral --controller=true --resource=true --namespaced=false --make=false
             $kb create webhook --group crew --version v1 --kind Admiral --defaulting
+	    $kb create api --group crew --version v1 --kind Laker --controller=true --resource=false --make=false
```

Comment:
Also see that it has make=false which means that is not calling the `make manifests` and it is not checking the change made.